files/goose-config.yaml

```diff
@@ -0,0 +1,20 @@
+GOOSE_PROVIDER: google
+GOOSE_MODEL: gemini-2.5-pro-preview-06-05
+extensions:
+  developer:
+    bundled: true
+    display_name: Developer
+    enabled: true
+    name: developer
+    timeout: 300
+    type: builtin
+  jiraextension:
+    bundled: null
+    description: null
+    enabled: true
+    env_keys: []
+    envs: {}
+    name: Jira Extension
+    timeout: 300
```

Comment:
Depending on the tasks at hand, the 5 minute timeout may be too strict.  I think it'll be high enough for Jira tasks but I still wanted to mention it here.

Reply:
I agree.  That's why I wrote it should be sufficient for the Jira task but for other operations probably not.

It was more meant as a general comment for us to look at the the timeouts and check which operations may take longer.